src/libraries/TickMath.sol

```diff
@@ -1,19 +1,24 @@
 // SPDX-License-Identifier: GPL-2.0-or-later
 pragma solidity ^0.8.0;
 
+import {CustomRevert} from "./CustomRevert.sol";
+
 /// @title Math library for computing sqrt prices from ticks and vice versa
 /// @notice Computes sqrt price for ticks of size 1.0001, i.e. sqrt(1.0001^tick) as fixed point Q64.96 numbers. Supports
 /// prices between 2**-128 and 2**128
 library TickMath {
+    using CustomRevert for bytes4;
+
     /// @notice Thrown when the tick passed to #getSqrtPriceAtTick is not between MIN_TICK and MAX_TICK
-    error InvalidTick();
+    error InvalidTick(int24 tick);
     /// @notice Thrown when the price passed to #getTickAtSqrtPrice does not correspond to a price between MIN_TICK and MAX_TICK
     error InvalidSqrtPrice();
```

Comment:
Why not also have a param here to be consistent with InvalidTick?

Reply:
oops this is another one like the currency one where i just didnt push the change to this definition, pushing